src/main/java/dk/kb/present/webservice/AccessUtil.java

```diff
@@ -35,6 +36,10 @@
 public class AccessUtil {
     private static final Logger log = LoggerFactory.getLogger(AccessUtil.class);
 
+    public static final String HEADER_SIMULATED_GROUP = "Simulated-OAuth2-Group";
+    public static final String GROUP_INTERNAL_SERVICE = "internal_service";
+    public static final String GROUP_ADMIN = "admin";
```

Comment:
Question: How have these values been chosen?

Reply:
By careful deliberation and pondering for about 5 seconds.

They will be replaced later on. I'll add a comment about it to the code.